scene/resources/packed_scene.cpp

```diff
@@ -527,7 +527,7 @@ Node *SceneState::instantiate(GenEditState p_edit_state) const {
 
 			bool valid;
 			Array array = dnp.base->get(dnp.property, &valid);
-			ERR_CONTINUE(!valid);
+			ERR_CONTINUE_EDMSG(!valid, vformat("Failed to get property '%s' from node '%s'.", dnp.property, dnp.base->get_name()));
```

Comment:
There's the same code in the `Variant::DICTIONARY` branch, would be good to use the same better message there too.

Reply:
Also just to be clear, the `EDMSG` variant of the macro means the error will be shown in the editor toaster. Is that what you intended? (i.e. is this an error that the user can trigger from the editor and where it would make sense to show in the toaster and not just in the Output dock?)

Reply:
Should I open a separate PR for that?

Reply:
No this is exactly the same change as done here, just handling Dictionary and not just Array, so it should be part of the same commit (you can edit it with `git commit --amend` and `git push --force`).